modules/common/src/main/scala/surge/core/Controllable.scala

```diff
@@ -17,7 +17,7 @@ trait ControllableRemover {
 trait Controllable {
```

Comment:
Hmm I'm not sure I'm crazy about `stopInternal` now that I'm seeing it in this change but maybe `Controllable` should be internal and we can expose an explicit `start`/`stop` from the javadsl and scaladsl.

Reply:
I think that would need  a discussion cause we have to change it for every method in the `contollable` trait. we van discuss in the stand up call